ccdImports/configFiles/CCD_Probate_Backoffice/FixedLists.json

```diff
@@ -130,11 +130,11 @@
   {"LiveFrom": "06/01/2017", "ID": "relationshipToDeceasedFixedList", "ListElementCode": "parent", "ListElement": "Parent"},
   {"LiveFrom": "06/01/2017", "ID": "relationshipToDeceasedFixedList", "ListElementCode": "other", "ListElement": "Other"},
   {"LiveFrom": "06/01/2017", "ID": "relationshipToDeceasedFixedList", "ListElementCode": "adoptedChild", "ListElement": "Adopted child"},
-  {"LiveFrom": "06/01/2017", "ID": "martialStatusFixedlList", "ListElementCode": "neverMarried", "ListElement": "Never married"},
-  {"LiveFrom": "06/01/2017", "ID": "martialStatusFixedlList", "ListElementCode": "widowed", "ListElement": "Widowed, their spouse or civil partner died before them"},
-  {"LiveFrom": "06/01/2017", "ID": "martialStatusFixedlList", "ListElementCode": "marriedCivilPartnership", "ListElement": "Married / in a civil partnership"},
-  {"LiveFrom": "06/01/2017", "ID": "martialStatusFixedlList", "ListElementCode": "divorcedCivilPartnership", "ListElement": "Divorced / civil partnership has dissolved"},
-  {"LiveFrom": "06/01/2017", "ID": "martialStatusFixedlList", "ListElementCode": "judicially", "ListElement": "Judicially seperated"},
+  {"LiveFrom": "06/01/2017", "ID": "martialStatusFixedList", "ListElementCode": "marriedCivilPartnership", "ListElement": "Married or in a civil partnership"},
```

Comment:
I think you can apply an ordering to this type of fixed list - so that drop down elements always appear in a set order